aspnetcore/signalr/version-differencs.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Differences between SignalR and ASP.NET Core SignalR
+author: rachelappel
+description: Differences between SignalR and ASP.NET Core SignalR 
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.date: 06/30/2018
+ms.prod: asp.net-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/version-differences
+---
+
+# Differences between SignalR and ASP.NET Core SignalR
```

Comment:
For clarity, change "SignalR" to "ASP.NET SignalR". Apply this change to the `description` metadata field too.